bin/cli.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+'use strict';
+
+require('codemod-cli').runTransform(
+  __dirname,
+  process.argv[2] /* transform name */,
+  process.argv.slice(3) /* paths or globs */
```

Comment:
To hard code the codemod name to use, we need to tweak this:

```suggestion
  'convert-htmlbars-import' /* transform name */,
  process.argv.slice(2) /* paths or globs */
```